owncloudApp/src/main/java/com/owncloud/android/utils/PermissionUtil.java

```diff
@@ -18,7 +18,7 @@
      * Determine whether <em>the app</em> has been granted a particular permission.
      *
      * @param permission The name of the permission being checked.
-     * @return <code>true</code> if app has the permission, or <code>false</code> if not.
+     * @return <code>false</code> if app has the permission, or <code>true</code> if not.
      */
     public static boolean checkSelfPermission(Context context, String permission) {
```

Comment:
The point here is that I changed the documentation because it was wrong, but the name of the function doesn't make sense anymore and I wasn't able to find a suitable name for it. The real fix here would be refactoring the body function so that it is consistent with the name, but it implies changes in every place where it is used. `isPermissionGranted` transmits the same meaning than the current `checkSelfPermission` in my opinion, so it wouldn't be a great change

Reply:
Ok, finally changed the name of the function to `isPermissionNotGranted`